project_internal_access_from_portal/tests/__init__.py

```
@@ -0,0 +1,3 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from . import test_portal_internal_access
```

Ruff (0.8.2)

`.test_portal_internal_access` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)